### PR TITLE
Add Cluster ID to About Modal

### PR DESCRIPTION
**Pull request by nicolethoen**
screen shot 2019-01-08 at 2 37 13 pm

### CODE REVIEW

File: frontend/public/clusters.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+export enum ClusterValues {
+  CLUSTER_ID = 'CLUSTER_ID'
+}
+
+const SET_CLUSTER_VALUE = 'setClusterValue';
+const DEFAULTS = _.mapValues(ClusterValues, undefined);
+export const setClusterValue = (name, value) => ({name, value, type: SET_CLUSTER_VALUE});
+
+export const clusterValueReducer = (state: ImmutableMap<string, any>, action) => {
```
Comment (spadgett) [inline frontend/public/clusters.ts]: I'd just add this to the existing ui-reducers and ui-actions. This seems too specific to be its own reducer.

File: frontend/public/components/_about-modal.scss [hunk 2]
```diff
@@ -2,8 +2,14 @@
   dt {
     font-size: var(--pf-c-content--FontSize);
     text-transform: none;
+    color: var(--pf-global--color--100);
+    margin-bottom: var(--pf-global--spacer--md);
```
Comment (spadgett) [inline frontend/public/components/_about-modal.scss]: alphabetize the rules

File: frontend/public/components/about-modal.jsx [hunk 3]
```diff
@@ -3,52 +3,61 @@ import { AboutModal as PfAboutModal, TextContent, TextList, TextListItem } from
...
+    }
+
+    _checkKubernetesVersion() {
+      k8sVersion()
+        .then(data => this.setState({ kubernetesVersion: data.gitVersion }))
+        .catch(() => this.setState({ kubernetesVersion: 'unknown' }));
+    }
+
+    _getClusterName() {
+      k8sVersion();
```
Comment (spadgett) [inline frontend/public/components/about-modal.jsx]: It's not clear to me why `_getClusterName` is calling `k8sVersion`

File: frontend/public/clusters.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import * as _ from 'lodash-es';
+import { Map as ImmutableMap } from 'immutable';
+
+export enum ClusterValues {
+  CLUSTER_ID = 'CLUSTER_ID'
+}
+
+const SET_CLUSTER_VALUE = 'setClusterValue';
+const DEFAULTS = _.mapValues(ClusterValues, undefined);
+export const setClusterValue = (name, value) => ({name, value, type: SET_CLUSTER_VALUE});
```
Comment (spadgett) [inline frontend/public/clusters.ts]: I don't think this needs to be a map of values. Just have a single action that sets a cluster ID.

**Comment by nicolethoen** [comment]
Here is an updated screen shot.

screen shot 2019-01-11 at 3 01 09 pm

**Comment by spadgett** [comment]
Thanks @nicolethoen 👍

**Comment by spadgett** [comment]
jenkins rebuild

**Pull request merged**